test/tests/serial/user.bats

```diff
@@ -0,0 +1,42 @@
+load "${REPOSITORY_ROOT}/test/helper/common"
+load "${REPOSITORY_ROOT}/test/helper/setup"
+
+BATS_TEST_NAME_PREFIX='[User] '
```

Comment:
This is probably not as descriptive as I'd like. Along with the test file name.

I also feel that way about the choice in ENV names. I think from the test failures we are only using this for Dovecot to run as a "vmail" user with static UID + GID for `/var/mail`.

```suggestion
BATS_TEST_NAME_PREFIX='[ENV] (DMS_VMAIL_UID + DMS_VMAIL_GID)'
```

**NOTE:** Changing the filename and ENV vars can be handled later, after you've got tests that actually pass properly.